header/color.h

```diff
@@ -1,6 +1,6 @@
 #define PLUM_PIXEL_INDEX(image, col, row, frame) (((size_t) frame * (size_t) (image) -> height + (size_t) row) * (size_t) (image) -> width + (size_t) col)
 
-#if PLUM_VLA_SUPPORT
+#ifdef PLUM_VLA_SUPPORT
```

Comment:
This is not identical to the original condition... I think the definition of `PLUM_VLA_SUPPORT` needs a rewrite instead.

Reply:
I see, it was doing `#define PLUM_VLA_SUPPORT (!defined(__STDC_NO_VLA__) || !(__STDC_NO_VLA__ + 0))`. I could rewrite it to replace `#define PLUM_VLA_SUPPORT (!defined(__STDC_NO_VLA__) || !(__STDC_NO_VLA__ + 0))` with `#if (!defined(__STDC_NO_VLA__) || !(__STDC_NO_VLA__ + 0)) \ #define PLUM_VLA_SUPPORT \ #endif`, and replace the two `#define PLUM_VLA_SUPPORT 0` with doing nothing. Sounds good?